server/src/main/java/org/apache/druid/segment/loading/SegmentLoaderLocalCacheManager.java

```diff
@@ -99,7 +99,10 @@ public SegmentLoaderLocalCacheManager(
           )
       );
     }
-    this.strategy = config.getStorageLocationSelectorStrategy(locations);
+
+    this.strategy = config.getStorageLocationSelectorStrategy();
+    this.strategy.setLocations(locations);
+    log.info("using storage location strategy: %s", this.strategy.getClass().getSimpleName());
```

Comment:
Generally, the first letter of the exception information needs to be capitalized. In addition, %s needs to be surrounded by square brackets. So please use `Using storage location strategy: [%s]`.

Reply:
Been fixed.